target/linux/ath79/dts/qca9563_tplink_eap2x5-1port.dtsi

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
```

Comment:
This include should be moved before the others.

Reply:
Fixed